src/Command/AddApproval/AddApprovalCommandHandler.cs

```diff
@@ -43,7 +43,8 @@ public async Task Handle(AddApprovalCommand command, CancellationToken cancellat
                 command.AccountLegalEntityId,
                 command.UKPRN,
                 command.EmployerAccountId,
-                command.TrainingCourseVersion);
+                command.TrainingCourseVersion,
+                false);
```

Comment:
It appears we never instantiate with PaymentsFrozen true, do we need this property or could we just set the status to false in the CreateNew method?

Reply:
yeah I was on the fence about this, in reality we will replace it with ApprenticeshipPriceEpisode very soon as per the design so I didn't spend much time overthinking it.

Reply:
However thinking about it when we do that replacement we definitely won't need an argument for it, will remove